### DISCUSSION TITLE
Metacell and scenic plus

### DISCUSSION THREAD

**Discussion by alexwang1001** [aertslab/scenicplus#145] [General]
Do you think metacell (aggregating reads from several single cells) could be a legitimate input for Scenic+? Is the sparse nature of the data necessary?
Thanks!

**Comment by robertzeibich** [comment]
@alexwang1001 Do you have any news? Have you used metacells output as scenic input?

**Reply by SeppeDeWinter** [reply]
Hi @robertzeibich 

Yes, it makes sense to use metacells for SCENIC+.

Best,

Seppe
